x/payment/keeper/stream_record.go

```diff
@@ -501,7 +468,7 @@ func (k Keeper) AutoResume(ctx sdk.Context) {
 			_, err := k.UpdateStreamRecordByAddr(ctx, flowChange)
 			if err != nil {
 				ctx.Logger().Error("auto resume, update receiver stream record failed", "address", outFlow.ToAddress, "err", err.Error())
-				break
+				panic("should not happen")
```

Comment:
It's better to print the real error msg when panic.

Reply:
because it will panic, we can get the stack trace clearly.